src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.FigureSupplier;
+
 /**
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
-
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figureArray = new Figure[5];
```

Comment:
```suggestion
        Figure[] figures = new Figure[5];
```

and lets make 5 as constant